courses/free-courses-pt_BR.md

```diff
@@ -243,6 +243,7 @@
 ### Node.js
 
 * [Criando APIs com NodeJs](https://www.youtube.com/playlist?list=PLHlHvK2lnJndvvycjBqQAbgEDqXxKLoqn) - Balta.io (YouTube)
+* [Curso de Node](https://www.youtube.com/watch?v=XN705pQeoyU&list=PLx4x_zx8csUjFC41ev2qX5dnr-0ThpoXE) - CFBCursos (YouTube)
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

Reply:
@eshellman This is a link to a playlist on youtube, just access it to view. And as it is presented in the contribution guide I write the name of the youtube channel. Give me another suggestion to improve.